dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapper.java

```diff
@@ -225,15 +231,23 @@ private void validateUpdateDurationParams(RequestParams requestParams)
     }
   }
 
-  // TODO Use RequestParamUtils.validateOrgUnitParams as soon /events accepts a list of org units
-  // UIDs
-  private void validateOrgUnitParams(UID orgUnit, OrganisationUnitSelectionMode orgUnitMode)
-      throws BadRequestException {
-    if (orgUnit != null && (orgUnitMode == ACCESSIBLE || orgUnitMode == CAPTURE)) {
+  private void validateOrgUnitMode(RequestParams params) throws BadRequestException {
+    if ((params.getOrgUnitMode().equals(ACCESSIBLE) || params.getOrgUnitMode().equals(CAPTURE))
```

Comment:
`params.getOrgUnitMode()` could be null at this point?
If yes, we could have an NPE here

Reply:
there's a condition on this method call, it's just called if orgUnitMode is not null.